cmd/operator-sdk/new/cmd.go

```diff
@@ -459,18 +458,16 @@ func getDeps() error {
 }
 
 func initGit() error {
-	if skipGit {
+	if !gitInit {
 		return nil
 	}
+	if _, err := exec.LookPath("git"); err != nil {
+		return errors.Wrapf(err, "failed to find git")
```

Comment:
Note that we're not silently skipping `git init` anymore if we can't find or run `git`. 
Since the user now opts-in for `git int` we should report the error if it fails instead of silently skipping.

Reply:
Makes sense! But maybe we can skip checking this as the next command will fail with an error that is self-descriptive, in my case it would be:
```
command not found: git
```